### PR TITLE
[ISSUE #2254] Method concatenates the result of a toString() call [CCSubClient]

### LINKED ISSUE #2254: [Enhancement] Method concatenates the result of a toString() call [CCSubClient]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
src/test/java/org/apache/eventmesh/runtime/demo/CCSubClient.java line 47
analysis and explanation:
This method concatenates the output of a toString() call into a StringBuffer or StringBuilder. It is simpler just to pass the object you want to append to the append call, as that form does not suffer the potential for NullPointerExceptions, and is easier to read.

### Describe the solution you'd like

reference:
image

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by momoxiaohaiya**
Fixes #2254.

### CODE REVIEW

**Comment by LIU-WEI-git** [comment]
Welcome to EventMesh😁 @momoxiaohaiya

**Comment by LIU-WEI-git** [comment]
Hi, thanks for your contribution. It seems code style has some problem. You could refer to https://eventmesh.apache.org/community/contribute#code-style

File: eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/CCSubClient.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public static void main(String[] args) throws Exception {
         subClient.registerBusiHandler(new ReceiveMsgHook() {
             @Override
             public void handle(Package msg, ChannelHandlerContext ctx) {
-                logger.error("Received message: -----------------------------------------" + msg.toString());
+                logger.error("Received message: {}", msg);
```
Comment (jonyangx) [inline eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/CCSubClient.java:47]: suggest implement Package class  toString() method.

Comment (momoxiaohaiya) [reply eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/CCSubClient.java:47]: thanks，i will try it

Comment (momoxiaohaiya) [reply eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/CCSubClient.java:47]: Package class has @Data annotation which has implemented toString() method.

**Review by LIU-WEI-git — APPROVED** [review]

**Pull request merged**